### PR TITLE
New referenceValue field Create Payment endpoint

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to include the field "referenceValue" in the Create Payment endpoint. (EDU-10947)

### CODE REVIEW

File: VTEX - Payment Provider Protocol.json [hunk 1]
```diff
@@ -867,36 +756,33 @@
...
                                                 "name": {
                                                     "type": "string",
-                                                    "description": "The custom field name."
+                                                    "description": "The custom field name.",
+                                                    "example": "Public Merchant ID Test."
                                                 },
                                                 "value": {
                                                     "type": "string",
-                                                    "description": "The custom field value."
+                                                    "description": "The custom field value.",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:764]:
```suggestion
                                                    "description": "Custom field value.",
```

File: VTEX - Payment Provider Protocol.json [hunk 2]
```diff
@@ -867,36 +756,33 @@
                                             "properties": {
                                                 "name": {
                                                     "type": "string",
-                                                    "description": "The custom field name."
+                                                    "description": "The custom field name.",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:759]:
```suggestion
                                                    "description": "Custom field name.",
```

File: VTEX - Payment Provider Protocol.json [hunk 3]
```diff
@@ -582,66 +515,49 @@
...
-                                                        "pattern": "^(.*)$"
+                                                        "example": "RJ"
                                                     }
                                                 }
                                             },
                                             "billingAddress": {
                                                 "type": "object",
-                                                "title": "billingAddress",
-                                                "description": "Billing address. Address of the payer.",
+                                                "description": "Buyer's billing address information.",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:560]: Para considerar no próximo PR: faria sentido substituir o termo **Buyer** por **Shopper** em todos os lugares em que ele aparece nessa API?

File: VTEX - Payment Provider Protocol.json [hunk 4]
```diff
@@ -482,87 +439,63 @@
...
-                                                        "title": "corporateDocument",
                                                         "description": "Buyer's corporate document number. This field is required if `isCorporate` is `true`.",
-                                                        "example": "123456789",
-                                                        "pattern": "^(.*)$"
+                                                        "example": "123456789"
                                                     },
                                                     "createdDate": {
                                                         "type": "string",
-                                                        "title": "createdDate",
                                                         "description": "Buyer's create datetime in format yyyy-MM-ddTHH:mm:ss.",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:497]:
```suggestion
                                                        "description": "Date when the buyer's profile was created in yyyy-MM-ddTHH:mm:ss format.",
```

File: VTEX - Payment Provider Protocol.json [hunk 5]
```diff
@@ -138,7 +138,7 @@
...
...4E20D3B4E07B7E871F5B5BC9F91\\\",\\\"timestamp\\\":\\\"2020-10-29 17:15:59Z\\\"}\"\r\n },\r\n \"nsu\": \"NSU-227AFD0BD2\",\r\n \"tid\": \"TID-DBE4BFFB19\",\r\n \"acquirer\": \"TestPay\",\r\n \"code\": \"2001\",\r\n \"message\": null,\r\n \"delayToAutoSettle\": 21600,\r\n \"delayToAutoSettleAfterAntifraud\": 1800,\r\n \"delayToCancel\": 21600\r\n}\r\n```\r\n\r\n### 200 - OK - Redirect\r\n\r\n```json\r\n{\r\n \"paymentId\": \"7ee64e51-a0d3-4405-874c-d7497ab84572\",\r\n \"status\": \"undefined\",\r\n \"tid\": \"214c699cb408ce6a7110\",\r\n \"paymentUrl\": \"http://php-connector.herokuapp.com/installments.php?paymentId=7ee64e51-a0d3-4405-874c-d7497ab84572\"\r\n}\r\n```\r\n\r\n### 200 - OK - Callback\r\n\r\n```json\r\n{\r\n \"paymentId\": \"7ee64e51-a0d3-4405-874c-d7497ab84572\",\r\n \"status\": \"approved\",\r\n \"authorizationId\": \"3baafb4097d6c8ad3883\",\r\n \"paymentUrl\": null,\r\n \"nsu\": \"214c699cb408ce6a7110\",\r\n \"tid\": \"214c699cb408ce6a7110\",\r\n \"acquirer\": null,\r\n \"code\": null,\r\n \"message\": \"Payment with custom installments approved\",\r\n \"delayToAutoSettle\": 21600,\r\n \"delayToAutoSettleAfterAntifraud\": 1800,\r\n \"delayToCancel\": 21600\r\n}\r\n```",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:141]: Acho que valeria mascarar o signature token no exemplo, para não expor isso.

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Payment Provider Protocol.json (hunk 3 above)
Comment (ricardoaerobr) [reply VTEX - Payment Provider Protocol.json:560]: Boa pergunta! Vou alinhar isso com o time de Payments!

File: VTEX - Payment Provider Protocol.json (hunk 5 above)
Comment (ricardoaerobr) [reply VTEX - Payment Provider Protocol.json]: Corrigido.

**Review by julia-rabello — APPROVED** [review]
LGTM

**Pull request merged**